core/src/mindustry/world/blocks/campaign/LaunchPad.java

```diff
@@ -98,7 +98,14 @@ public void updateTile(){
             if(!state.isCampaign()) return;
 
             //launch when full and base conditions are met
-            if(items.total() >= itemCapacity && efficiency() >= 1f && timer(timerLaunch, launchTime / timeScale)){
+            boolean launchable = true;
+            for(Item item : content.items()) {
+                if(state.secinfo.getRealDestination().save.meta.secinfo.coreItems.get(item) + items.get(item) > state.secinfo.getRealDestination().save.meta.secinfo.storageCapacity) {
```

Comment:
This would mean launch pads stop sending when **any** resource they have goes over the limit, which may not be desirable.